src/components/GameView/TargetSelectionOverlay.vue

```diff
@@ -18,7 +18,7 @@
         Cancel
       </v-btn>
     </div>
-    <v-btn icon data-cy="cancel-target-mobile" @click="$emit('cancel')">
+    <v-btn icon data-cy="cancel-target-mobile" variant=“text” @click="$emit('cancel')">
```

Comment:
How does this one look now? Could you add a screenshot of the new result? I'm wondering if we should change the color in addition to the button style or if it's good as-is. What do you think?